### ISSUE TITLE
Empty list when pagination is turned on

### ISSUE THREAD

**Issue by agte** [feathersjs-ecosystem/feathers-vuex#430]
Simple lists work fine. But if I turn pagination on ("paginate: true") then no data are displayed.
```
const { items: games } = useFind({
  model: context.root.$FeathersVuex.api.Game,
  params: reactive({
    paginate: true,
    qid: 'myGames',
    query: {
      $limit: 50,
      $skip: 0,
    },
  }),
});
```
I found one problem here: https://github.com/feathersjs-ecosystem/feathers-vuex/blob/master/src/useFind.ts#L106

```
const pagination =
  computes.paginationData[getterParams.qid || state.qid] || {}
...
const items = getItemsFromQueryInfo(
  pagination,
```
"paginationData" is computed entity, so it's a ref object and stores its data in "value" property, not directly. That's why "pagination" always is an empty object and then "items" is always an empty array.

```
computes.paginationData.value[getterParams.qid || state.qid] || {}
```
When I change this line, data are displayed at least.

**Comment by marshallswain** [comment]
Ah.  Good catch!  I'll patch this up.

**Comment by marshallswain** [comment]
Fixed in `feathers-vuex@3.7.0`.
